### ISSUE TITLE
[Bug] Physics asset not saving correctly

### ISSUE THREAD

**Issue by RenderingFiend** [turanszkij/WickedEngine#29]
Hi,
To do some testing I frequently like to make changes to Sponza. When I save my changes, everything saves correctly, except for the blowing curtain. When opening the changed wiscene file, the blowing flag seemingly disappears.
I ended up finding it: it became microscopic and is in the center of the floor, with no physics applied.
I'm assuming this isn't scene specific.

**Comment by turanszkij** [comment]
Ah yes, I know this bug. 

There is a slight problem that I need to find a solution for that makes sense: The physics mesh for soft bodies is created for the mesh in its "non-simulated pose", basically this is the mesh without any physics applied. The soft body physics is modifying vertices of the mesh, so it will no longer will be in the "non-simulated pose". If it's saved in the simulated pose, then when loaded, the soft body will be created again using the already simulated pose and results will be *very* different and not stable.

TLDR: Never the less, the current state is bad and it will be fixed, I will put a priority on it. For the time being, a workaround is to disable physics (there is an Editor checkbox for this on the right hand side) before loading the scene and saving it.

**Comment by RenderingFiend** [comment]
Awesome, thanks. For now I'll just use the workaround you mentioned. Keep up the great work. I'm finding Wicked Engine to be my favorite game engine (UE4 is too monolithic, Unity is a pain to learn).

**Comment by turanszkij** [comment]
Fixed by having the soft body rest pose serialized. The save file size will increase a bit. Soft body shape will be preserved correctly after saving and loading. Let me know if there are issues with this. Thanks for reporting!
